playbooks/02 - Use Case - Asset Change Activity/Manage Asset Change Activity Closure.json

```diff
@@ -97,13 +193,43 @@
     "routes": [
         {
             "@type": "WorkflowRoute",
-            "name": "Start -> Update Asset Change Activity",
+            "name": "Get All Incomplete Task -> Is Incomplete Task Exists",
+            "targetStep": "\/api\/3\/workflow_steps\/110145ac-6461-473f-b3c5-d82ab529d4ce",
+            "sourceStep": "\/api\/3\/workflow_steps\/6f263d5c-06e0-47ff-8a3e-99867c8ad347",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "befccc10-e4fc-4725-8950-d6719061559c"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Is Incomplete Task Exists -> Copy of Update Asset Change Activity",
```

Comment:
Copy of Update Asset Change Activity -->> ??

Reply:
incorporated